tests/e2e/e2e_test.go

```diff
@@ -116,6 +124,70 @@ var _ = ginkgo.AfterSuite(func() {
 })
 
 var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
+	ginkgo.It("can start with blockchain", func() {
```

Comment:
probably call this "can create blockchains"

Reply:
I don't know this should be the first test in the list. it is somewhat advanced functionality. maybe execute it after testing basic stuff (start newtork/create node/remove node/etc)

Reply:
Well in CI it doesn't really matter what comes first, failure is failure. I did put this at the start because I wanted to avoid that later there might be issues because of existing state.